clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -28,37 +28,37 @@
  * This is applicable when the consumer is having Kafka auto-manage group membership. If the consumer directly assigns partitions,
  * those partitions will never be reassigned and this callback is not applicable.
  * <p>
- * When Kafka is managing the group membership, a partition re-assignment will be triggered any time the members of the group change or the subscription
+ * When Kafka is managing the group membership, a partition re-assignment will be triggered whenever the members of the group change or the subscription
  * of the members changes. This can occur when processes die, new process instances are added or old instances come back to life after failure.
- * Partition re-assignments can also be triggered by changes affecting the subscribed topics (e.g. when the number of partitions is
+ * Partition re-assignments can also be triggered by changing the subscribed topics (e.g. when the number of partitions is
  * administratively adjusted).
  * <p>
  * There are many uses for this functionality. One common use is saving offsets in a custom store. By saving offsets in
  * the {@link #onPartitionsRevoked(Collection)} call we can ensure that any time partition assignment changes
  * the offset gets saved.
  * <p>
  * Another use is flushing out any kind of cache of intermediate results the consumer may be keeping. For example,
- * consider a case where the consumer is subscribed to a topic containing user page views, and the goal is to count the
- * number of page views per user for each five minute window. Let's say the topic is partitioned by the user id so that
+ * consider a case where the consumer is subscribing to a topic containing user page views, and the goal is to count the
+ * number of page views per user for each five minutes window. Let's say the topic is partitioned by the user id so that
```

Comment:
"minute" should be fine here because it's part of a modifier for "window" ("five-minute window" if we want to be really accurate about it)